main/sw/inc/shellio.hxx

```diff
@@ -501,32 +497,32 @@ public:
 
     const String&       GetBaseURL() const { return sBaseURL;}
 
-    // suche die naechste Bookmark-Position aus der Bookmark-Tabelle
+    // search next bookmark position from the bookmark table
 	sal_Int32 FindPos_Bkmk( const SwPosition& rPos ) const;
 	// build a bookmark table, which is sort by the node position. The
 	// OtherPos of the bookmarks also inserted.
 	void CreateBookmarkTbl();
-	// search alle Bookmarks in the range and return it in the Array
+	// search all bookmarks in the range and return it in the Array
 	sal_uInt16 GetBookmarks( const SwCntntNode& rNd,
 						xub_StrLen nStt, xub_StrLen nEnd,
 						SvPtrarr& rArr );
 
-	// lege einen neuen PaM an der Position an
+	// create a new PaM at the position
     static SwPaM * NewSwPaM(SwDoc & rDoc,
                             sal_uLong const nStartIdx, sal_uLong const nEndIdx);
 
-	// kopiere ggfs. eine lokale Datei ins Internet
+	// upload a local file to the internet / cloud
 	sal_Bool CopyLocalFileToINet( String& rFileNm );
 
-	// Stream-spezifische Routinen, im Storage-Writer NICHT VERWENDEN!
+	// Stream-specific routines, DO NOT USE IN Storage-Writer!
 
-	// Optimierung der Ausgabe auf den Stream.
+	// Optimize of the output stream
 	SvStream& OutLong( SvStream& rStrm, long nVal );
 	SvStream& OutULong( SvStream& rStrm, sal_uLong nVal );
 
-	// Hex-Zahl ausgeben, default ist 2.stellige Zahl
+	// print Hex-Zahl, default is 2. digit number
```

Comment:
Hex-Zahl could be translated into "hexadecimal number" unless Hex-Zahl is an actual class or type name

Reply:
Ohh man I missed that. :facepalm: